Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.14.0a3-slim
+LABEL maintainer="yaroslavbordovoy@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR app/
```

Comment:
The `WORKDIR` path should not end with a slash. It should be `WORKDIR app` instead of `WORKDIR app/`.

Reply:
Nothing will change if I remove /